modules/pvr/texture_loader_pvr.cpp

```diff
@@ -215,12 +216,10 @@ static void _compress_pvrtc4(Image *p_img) {
 			int ofs, size, w, h;
 			img->get_mipmap_offset_size_and_dimensions(i, ofs, size, w, h);
 			Javelin::RgbaBitmap bm(w, h);
-			copymem(bm.GetData(), &r[ofs], size);
-			{
+			for (unsigned i = 0; i < size / 4; i++) {
```

Comment:
Should use another letter, this is already in a `for (int i)` loop.

Reply:
Stupid mistake :+1: I would expect we as GCC can produce a warning for it..

Reply:
It does with ```-Wshadow```

Reply:
You do, that's how I found about it :)
https://travis-ci.org/godotengine/godot/jobs/484316612